include/tvm/runtime/c_runtime_api.h

```diff
@@ -76,6 +76,18 @@ extern "C" {
 #endif
 #include <stddef.h>
 #include <stdint.h>
+#include <stdio.h>
+#include <sys/types.h>
```

Comment:
@mkatanbaf @mehrdadh @areusch why is this header needed ?
Is it just because to use the type ssize_t ?

Reply:
Also do we want this added to c_runtime_api ? 
(As this is affecting bare metal deployments)

Reply:
@manupa-arm @areusch The <stdio.h> is needed to define ssize_t (not on windows, for windows we directly used int64_t and int32_t instead of ssize_t). The <sys/type.h> was needed on hexagon.
We discussed the proper place to add this definition and landed on the c_runtime_api. @manupa-arm please lmk if you have another suggestion.

Reply:
@manupa-arm  thanks. there was an issue with ssize_t being defined in other parts of the code which caused a redefinition error. our plan was to unify the different definitions in a subsequent PR soon. I'll check if I can find a better place for the definition in order to minimize the effect on bare metal deployments.

Reply:
perhaps we can move this to e.g. `tvm/support/types.h` or something of that nature.